api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -1157,6 +1175,26 @@ public ScheduledExecutorService getScheduledExecutorService() {
      */
     public abstract String getAuthority();
 
+    /**
+     * Returns the ChannelCredentials used to construct the channel, without bearer tokens.
+     *
+     * @since 1.35.0
+     */
+    public ChannelCredentials getChannelCredentials() {
+      throw new UnsupportedOperationException();
```

Comment:
Feel free to implement this now via `getUnsafeChannelCredentials().withoutBearerTokens()`